NitroxClient/MonoBehaviours/PlayerMovementBroadcaster.cs

```diff
@@ -27,13 +27,10 @@ public void Awake()
 
     public void FixedUpdate()
     {
-        // Throttle location broadcasts to not run on every physics tick.
-        if (locationBroadcastSkipThreshold-- > 0)
+        if (Time.fixedTime >= nextLocationBroadcast)
         {
-            return;
+            nextLocationBroadcast = Time.fixedTime + LOCATION_BROADCAST_TIME;
         }
-        // Reset skip threshold.
-        locationBroadcastSkipThreshold = LOCATION_BROADCAST_TICK_SKIPS;
```

Comment:
The return functionality is lost now, isn't it? So the timing code has no effect?